src/Element/Os2formsPersonLookup.php

```diff
@@ -18,12 +18,12 @@ public static function getCompositeElements(array $element) {
     $elements = [];
     $elements['cpr_number'] = [
       '#type' => 'textfield',
-      '#title' => t('CPR number'),
+      '#title' => t('CPR nummer'),
```

Comment:
In Danish this is spelled with a dash, i.e. “CPR-nummer” (many native Danish writers don't know this).

Reply:
Fixed